src/Common/src/Interop/Unix/libc/Interop.getsetpriority.cs

```diff
@@ -1,23 +1,48 @@
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+using System.Diagnostics;
 using System.Runtime.InteropServices;
+using System.Reflection;
 
 internal static partial class Interop
 {
     internal static partial class libc
     {
         [DllImport(Libraries.Libc, SetLastError = true)]
-        internal static extern int getpriority(PriorityWhich which, int who);
+        private static extern int getpriority(PriorityWhich which, int who);
+
+        internal static int getpriority(PriorityWhich which, int who, out int priority)
+        {
+            priority = getpriority(which, who);
+            return Marshal.GetLastWin32Error();
```

Comment:
I think it'd be worth a comment explaining why this is correct, even though it looks strange.